### PR TITLE
Added building strategy plan

### PR DESCRIPTION
**Pull request by MichielvdBerg**
A short description of the building strategy

User story:
"As an agent
When in a simulation
I am able to build stuff if I think this 
is benificial for me.
To optimize a certain indicator or to 
reach a construction goal."

### CODE REVIEW

File: documents/Strategies/Building strategy.tex [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+this is benificial for me\\
+To optimize a certain indicator."\\}
+
+\section{When/why to build buildings}
+In this section the reasons to build a building will be explained. The first reason to build a building, is that it can influence an indicator in a positive way. Another reason why to build something, is that a building can give you profit in terms of your budget.
+
+
+\subsection{Direct influence}
+There is one direct reason to build a building. We created an indicator that has the goal to make high buildings. The more high buildings, the better the indicator will be. But there is a downside with building something. Building a building costs a lot of money and the budget is not unlimited. The agent will build stuff as long as the budget holds and as long as it gives a positive influence on the indicators.
+
```
Comment (jzoon) [inline documents/Strategies/Building strategy.tex:30]: Building new buildings usually has a positive effect on the budget, so this isn't a real problem. Only destroying buildings costs a lot of money.

Also, there are two building indicators. One of them just needs more Education space, and the other one also needs high buildings. Maybe write something about this in this paragraph.

File: documents/Strategies/Building strategy.tex [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+\section{Which building to build}
+This section explains which building to build. Because our vision is to build high buildings, the buildings that are build, needs to be high buildings. In this way, it is more beneficial for the indicators. So when there is ground left, a high building is built on that ground if it beneficial for the indicators.
+
```
Comment (jzoon) [inline documents/Strategies/Building strategy.tex]: The high buildings the agent should build are of the type 'Luxe variant'. I think you should add this to this paragraph. Also, if the high buildings indicator is at 100%, it doesn't matter which education building to build.

Comment (MichielvdBerg) [inline documents/Strategies/Building strategy.tex]: Good point, I have improved this feedback

File: documents/Strategies/Building strategy.tex [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+I am able to build stuff if I think 
+this is benificial for me\\
+To optimize a certain indicator."\\}
+
+\section{When/why to build buildings}
+In this section the reasons to build a building will be explained. The first reason to build a building, is that it can influence an indicator in a positive way. Another reason why to build something, is that a building can give you profit in terms of your budget.
+
+
+\subsection{Direct influence}
+There is one direct reason to build a building. We created an indicator that has the goal to make high buildings. The more high buildings, the better the indicator will be. Another indicator we have, is the extension of education space. So to achieve this we need to build more education building. The agent will build stuff as long as the budget holds and as long as it gives a positive influence on the indicators.
```
Comment (daanwerf) [inline documents/Strategies/Building strategy.tex]: *buildings, in the third line and the word "stuff" may be a bit too vague of a term.

File: documents/Strategies/Building strategy.tex [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+
+\section{Which building to build}
+This section explains which building to build. Because a part of our vision is to build high buildings, the buildings that are build, needs to be high buildings. In the Tygron engine, high buildings are of the type “Luxe variant”, so these buildings needs to be built until the indicator has reached 100\%. When the indicator has reached 100\%, it does not matter which buildings are built for the high building indicator. For the extension of the education space, it does not matter which building is built, as long as the building is the most beneficial for the indicator. 
+
```
Comment (daanwerf) [inline documents/Strategies/Building strategy.tex:34]: Keep in mind that an indicator probably won't reach 100% in a regular simulation run.

Comment (MichielvdBerg) [inline documents/Strategies/Building strategy.tex:34]: That is true, but the goal is to reach 100, so when it's not, there needs to be more high buildings

**Comment by spiderbiggen** [comment]
This looks good.

**Pull request merged**